src/org/usfirst/frc/team2335/robot/subsystems/Vision.java

```diff
@@ -16,7 +16,7 @@
 	//This is the distance we want to be from the target it feet
 	//TODO: Find appropriate distance from target
 	private final float wantedDistanceInFeet = 12.0f;
-	
+
     public float getDistance()
     {
     	//Use floats, they're way more accurate (AKA: doubles are the devil)
```

Comment:
Why is nothing shown here? It this actually the whole Vision subsystem or is it only showing the parts changed?